prac_06/REFLECTION.md

```diff
@@ -0,0 +1,28 @@
+# CP1404 Practical Reflection
+
+Write short but thoughtful answers to each of the following.  
+Replace each `...` with your meaningful answer.
+
+## What have you changed about your process in this subject based on your personal reflection?
+
+I have actually started using a "TO-DO" app to organise what work
+has to get done for each subject, and by what date.
+
+## What do you still need to change in terms of your approach, application, time on task, attitude, etc.?
+
+I usually complete watching the lectures by the time the seminars are held; so I have not been
```

Comment:
Nice work with watching all the videos, don't be afraid to come to the seminars even just to listen to the questions of others, I know its a bit intimidating but it may be less daunting than you think :)